control/checkbox.go

```diff
@@ -0,0 +1,45 @@
+package control
+
+import (
+	"github.com/jbowtie/gokogiri/xml"
+	"strings"
+)
+
+const checkBoxAttributeKey = "val"
```

Comment:
Did you happen to find somewhere that describes the schema of checkbox fields, or did you just look at the XML and figure it out? I'm searching around and can't find the former to link to from here 😕